instagram/models.py

```diff
@@ -81,10 +81,10 @@ def object_from_dictionary(cls, entry):
         for version, version_info in six.iteritems(entry['images']):
             new_media.images[version] = Image.object_from_dictionary(version_info)
 
-        if new_media.type == 'video':
-            new_media.videos = {}
-            for version, version_info in six.iteritems(entry['videos']):
-                new_media.videos[version] = Video.object_from_dictionary(version_info)
+        # if new_media.type == 'video':
+        #     new_media.videos = {}
+        #     for version, version_info in six.iteritems(entry['videos']):
```

Comment:
Yeah, you're dealing with their object models from their API, hence why the dot notation to get it. More and more reasons we should just RYO an instagram http wrapper that just serializes json